langchain/evaluation/run_evaluators/string_run_evaluator.py

```diff
@@ -383,3 +384,40 @@ def from_model_and_evaluator(
             example_mapper=example_mapper,
             string_evaluator=evaluator,
         )
+
+    @classmethod
+    def from_data_type(
```

Comment:
Weird rn because datatype just refers to the dataset but the run mapper has everything to do with the model's runs - I think I should use something other than this enum (maybe `is_llm` bool? though that's restrictive)